src/testing_system/internal/usecase/Runner.py

```
@@ -0,0 +1,144 @@
+        Processing every single query, evaluates 
+
+        Usage example:
+        runner = Runner(...,
+                num_process_workers=1,
+                num_eval_workers=1,
+                use_threads=True) 
+        await runner.run(experiment)
+        """
+        
+        experiment_from_registry = self._exist(experiment=experiment)
+        if experiment_from_registry != None:
```

Ruff (0.15.10)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)